config-xml/addon/src/main/java/org/jboss/windup/config/parser/xml/PhaseHandler.java

```diff
@@ -29,7 +29,7 @@ public Void processElement(ParserContext context, Element element) throws Config
         Class<? extends RulePhase> phase = phaseFinder.findPhase(phaseStr);
         if (phase == null)
         {
-            String phasesListing = phaseFinder.getAvailablePhases().stream().map(clazz -> clazz.getSimpleName()).collect(Collectors.joining("\n    "));
```

Comment:
It looks like this one changes it without keeping the additional spaces. Is that intentional?

Reply:
probably not. I will fix it just for sure.